### ISSUE TITLE
"grid of blobs" after ts_reconstruct in WarpTools tutorial in spanish locale 

### ISSUE THREAD

**Issue by pconesa** [warpem/warp#265]
![image](https://github.com/user-attachments/assets/f25dca45-ea03-43cb-b669-fab26d0db0df)

**Comment by pconesa** [comment]
To avoid double posting I close this. I've asked the same thing in the google group. Maybe someone else knows.

**Comment by alisterburt** [comment]
just tested and runnning fine here on the main branch

```sh
WarpTools ts_reconstruct \
> --settings warp_tiltseries.settings \
> --angpix 10 \
> --perdevice 4 \
> --output_processing recon_test
```
![image](https://github.com/user-attachments/assets/bbcebc7c-3d61-4c64-91a4-3376f1bfba0c)

here is what the xml looks like for TS_1

[TS_1.xml](https://github.com/user-attachments/files/17247937/TS_1.xml.txt)


let's track here - easier than the mailing list

**Comment by alisterburt** [comment]
my suspects would be
- something wrong with tilt series alignment data (did `ts_aretomo` or `ts_etomo_*` run ok?)
- something wrong with tilt series ctf data (did you run `ts_ctf`?)

**Comment by pconesa** [comment]
Yes, we run both, we also saw that TS_01.xml changed the shifts values from something reasonable after ts_etomo to something suspiciously wrong after ts_ctf.

Can paste the values but from #.some decimals values changes to "huge integers"

I'll ask my colleague to attach the files

**Comment by pconesa** [comment]
We will make a quick test... but it may be a "locale" issue...? Machine is in ES (Spanish)... maybe we lost the decimal places due to decimal separator being different.

**Comment by alisterburt** [comment]
That would make sense, we've had a couple of locale related bugs - thought we had squashed them all but there might be some left over

**Comment by pconesa** [comment]
Can confirm it was a locale (ES) issue.

**Comment by alisterburt** [comment]
Thanks for confirming @pconesa - will leave open so we know to look into/fix it ☺️
